runner/execer/os/os_execer_test.go

```diff
@@ -148,7 +176,7 @@ func TestAbortCatch(t *testing.T) {
 	}
 
 	proc.Abort()
-	usage, err = e.memUsage(pid)
+	usage, _ = e.memUsage(pid)
```

Comment:
should we check this err

Reply:
I removed the call e.memUsage(pid).... it was not needed.